scheduler/src/cook/mesos/api.clj

```diff
@@ -669,6 +737,15 @@
     (ANY "/running" []
          (running-jobs conn))
 
+    ;; Resource shares:
+    (GET  "/resources/" request (get-resources conn request))
+    (GET  "/resources/users/:username" [username :as request] (get-user-resources conn request username))
+    (POST "/resources/users/:username" [username :as request] (set-user-resources! conn request username
```

Comment:
For the /resource/user endpoints I would prefer calling it share as it is the term used throughout the codebase so something like /share/user/:username